src/main/java/com/iri/dao3/controller/RenderAllCarsController.java

```diff
@@ -0,0 +1,25 @@
+package com.iri.dao3.controller;
+
+import com.iri.dao3.lib.Injector;
+import com.iri.dao3.model.Car;
+import com.iri.dao3.service.CarService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class RenderAllCarsController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllCarsController extends HttpServlet {
```
same for other models

Reply:
done